onnxruntime/test/providers/cpu/tensor/upsample_op_test.cc

```diff
@@ -692,7 +692,7 @@ TEST(UpsampleOpTest, NhwcUpsampleOp4D1CBilinearTest) {
   // TensorRT: results mismatch
   // ROCm: results mismatch
   test.Run(OpTester::ExpectResult::kExpectSuccess, "",
-           {kCudaExecutionProvider, kTensorrtExecutionProvider, kRocmExecutionProvider});
+           {kCudaExecutionProvider, kCudaNHWCExecutionProvider, kTensorrtExecutionProvider, kRocmExecutionProvider});
```

Comment:
If we are going through this approach for testing - should we get rid of the tests (or bring them over here) in tests/providers/cuda/nhwc ?

Reply:
The CPU tests tests against all backends except for the excluded ones - and those have been excluded for a reason. We should do the reverse, move cuda/nhwc tests over to the cpu tests for test cases which are not yet covered and remove the specialized nhwc tests.

Reply:
I agree. We should "unify" (i..e) pick up tests from the nhwc tests over to the cpu tests (after de-deuplication) and get rid of the nhwc tests.